crates/modelardb_compression/src/lib.rs

```diff
@@ -233,7 +233,7 @@ fn can_models_be_merged(
 }
 
 /// A compressed segment being built from an uncompressed segment using the
-/// model types in [`models`](crate::models). Each of the model types is used to
+/// model types in [`models`](models). Each of the model types is used to
```

Comment:
Maybe remove "(models)" as it should not be necessary [for rustdoc to create the correct link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html).

Reply:
Fixed.